build/common.sh

```diff
@@ -274,15 +249,15 @@ function kube::build::ensure_docker_daemon_connectivity {
       echo
       echo "Possible causes:"
       echo "  - On Mac OS X, DOCKER_HOST hasn't been set. You may need to: "
-      echo "    - Create and start your VM using docker-machine or boot2docker: "
-      echo "      - docker-machine create -d ${DOCKER_MACHINE_DRIVER} ${DOCKER_MACHINE_NAME}"
-      echo "      - boot2docker init && boot2docker start"
-      echo "    - Set your environment variables using: "
-      echo "      - eval \$(docker-machine env ${DOCKER_MACHINE_NAME})"
-      echo "      - \$(boot2docker shellinit)"
-      echo "    - Update your Docker VM"
-      echo "      - Error Message: 'Error response from daemon: client is newer than server (...)' "
-      echo "      - docker-machine upgrade ${DOCKER_MACHINE_NAME}"
+      echo "    - Set up Docker for Mac (https://docs.docker.com/docker-for-mac/)"
+      echo "    - Or, set up docker-machine"
+      echo "      - Create and start your VM using docker-machine: "
+      echo "        - docker-machine create -d ${DOCKER_MACHINE_DRIVER} ${DOCKER_MACHINE_NAME}"
+      echo "      - Set your environment variables using: "
+      echo "        - eval \$(docker-machine env ${DOCKER_MACHINE_NAME})"
+      echo "      - Update your Docker VM"
```

Comment:
I don't agree with this indentation change.
While the `upgrade` text is docker-machine specific, the 'update your docker vm' case is not a sub section of the 'set up docker-machine' bullet.

Alternatively, clarify the text with something like "If you see the Error Message ....  your docker is out of date."

I'm not sure how much discussion helptext we should have here. Possibly it should be a link to some 'basic development environment troubleshooting'.